src/Entity.d.ts

```diff
@@ -52,7 +52,7 @@ export class Entity {
    */
   getMutableComponent<C extends Component<any>>(
     Component: ComponentConstructor<C>
-  ): C;
+  ): C | undefined;
```

Comment:
Do you think this will be annoying? Should these functions throw an error (in development mode?) instead if the component does not exist?

Reply:
I think it's valuable to return `undefined` so you could check in your code if it exist or not instead of catching an error